frontend/app/src/components/settings/FrontendSettings.vue

```diff
@@ -162,360 +263,126 @@
   </setting-category>
 </template>
 
-<script lang="ts">
+<script setup lang="ts">
```

Comment:
We have to start using it more don't we?

Reply:
Yes, but I have problems with defineProps, so we can use for component without props

Reply:
And yeah 1 more problem, with this setup, LGTM.com don't know if we really used the imported components or not, so it throws warnings.